### PR TITLE
Additional cucumbers

### PR DESCRIPTION
**Pull request by BenHand**
testing search fields

### CODE REVIEW

File: app/controllers/rooms_controller.rb [hunk 1]
```diff
@@ -46,10 +46,10 @@ def create
       if @room.save
          redirect_to @room, notice: "#{@room.name} has been created"
        else
-         redirect_to :back, alert: "Error occured, room not saved"
+         redirect_to root_path, alert: "Error occured, room not saved"
```
Comment (JMD1986) [inline app/controllers/rooms_controller.rb:49]: :+1:

File: features/search.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    And I search for "frank"
+    Then I should see "Frank"
+
+  Scenario: I visit the meetings page and search
+    And I navigate to "/meetings"
+    And I search for "synergy"
+    Then I should see "Synergy"
+
+  # Cannot be used until one search bar on rooms page.
+  # Scenario: I visit the rooms page and search
```
Comment (jah2488) [inline features/search.feature:20]: We can give each of the search bars an ID and then in the step we can look them up based on their ID tag.

File: features/step_definitions/search.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+When(/^I search for "(.*?)"$/) do |query|
+  fill_in 'search', with: query
```
Comment (jah2488) [inline features/step_definitions/search.rb:2]: For example: `fill_in 'mainSearch', with: query`

**Comment by mlgiardina** [comment]
:+1:

**Comment by JMD1986** [comment]
:+1: :fire: :fire: :fire:

**Pull request merged**